src/java/org/apache/cassandra/db/partitions/PartitionUpdate.java

```diff
@@ -624,6 +626,8 @@ public void validateIndexedColumns()
     {
         public void serialize(PartitionUpdate update, DataOutputPlus out, int version) throws IOException
         {
+            Preconditions.checkArgument(version != MessagingService.VERSION_DSE_68,
```

Comment:
why? also if serializing to 6.8 we should probably serialize that maxTimestamp (or at least some long value), shouldn't we?

Reply:
exactly, "why?" is the question.
serializing max timestamp is not an issue really. the question is do we want to be able to serialize to 6.8...

Reply:
FWIW dse is able to deserialize from an older version and set the maxTimestamp itself

http://github.com/riptano/bdp/blob/15fc8132761f46a055f221cc619d8cc38b812630/dse-db/src/java/org/apache/cassandra/db/partitions/PartitionUpdate.java#L638-L639

Reply:
The rationale is as follows:
1. we want to deserialize 6.8 because of on-disk format
2. we don't want to serialize to 6.8 because if we talk to another dse node we should not negotiate using 6.8 format